Test.java

```diff
@@ -0,0 +1,35 @@
+import java.io.IOException;
+
+
+public class Test {
+
+	 private static int count = 0;
+
+	
+	public static void main(String[] args) {
+		
+				System.out.println("This is Gaurav Garg testing using GIT");
+				
+				System.out.println("Tetsing PR 3");
+																System.out.println("Tetsing PR");
+								System.out.println("Tetsing PR222");System.out.println("Tetsing PR");
+								System.out.println("Tetsing PR 35555");
+								System.out.println("Tetsing PR"); System.out.println("Tetsing PR1");
+				Process process =null;
+		
+			//System.out.println("This is Gaurav Garg testing using GIT");		
+				try {
+					process.getOutputStream().close();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "process" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS2259)